src/qt/bitcoin.cpp

```diff
@@ -398,8 +399,7 @@ void BitcoinApplication::initializeResult(bool success, interfaces::BlockAndHead
     qDebug() << __func__ << ": Initialization result: " << success;
 
     // Set exit result.
```

Comment:
In commit "gui: return EXIT_FAILURE on post-init fatal errors" (c8806e972f2be9f8a05bf11b1f6a8a2e70e01962)

Should delete the "Set exit result" comment here or move it down below. Probably fine to delete because `exit_status.store(EXIT_FAILURE)` should pretty clear by itself.

Reply:
done, deleted